packages/core/src/util/renderer.ts

```diff
@@ -172,6 +172,14 @@ export interface WithClassname {
   className?: string;
 }
 
+export interface EnumOption {
+  label: string;
+  description: string;
```

Comment:
Not sure what `description` is intended for. If we want to keep it we should probably make it optional.